### PR TITLE
Update issue template

### PR DESCRIPTION
**Pull request by akimd**
Related to https://github.com/docker/desktop-experience-backlog/issues/37.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
-Please check the [FAQ](FAQ.md).  Issues without logs and details
-cannot be exploited, and will be closed.
+Issues without logs and details cannot be exploited, and will be closed.
+
+  - [ ] I have tried with the latest version of my channel (Stable or Edge)
```
Comment (gtardif) [inline .github/ISSUE_TEMPLATE.md]: Not sure if users will easily conform to this (Stable versions in particular might not want to upgrade just to test, even though we would like them)

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
-Please check the [FAQ](FAQ.md).  Issues without logs and details
```
[the remark this answers is not in the document]
Comment (akimd) [reply .github/ISSUE_TEMPLATE.md]: yes, definitely.

[the remark this answers is not in the document]
Comment (akimd) [reply .github/ISSUE_TEMPLATE.md]: But I meant first to fix the broken link, and then take more time for the rest.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (akimd) [reply .github/ISSUE_TEMPLATE.md]: Still, I think we should promote updating.  Do you feel I should remove that item?

**Pull request merged**